tests/e2e/configurer/config/constants.go

```diff
@@ -6,9 +6,9 @@ const (
 	// if not skipping upgrade, how many blocks we allow for fork to run pre upgrade state creation
 	ForkHeightPreUpgradeOffset int64 = 60
 	// estimated number of blocks it takes to submit for a proposal
-	PropSubmitBlocks float32 = 10
+	PropSubmitBlocks float32 = 1
 	// estimated number of blocks it takes to deposit for a proposal
-	PropDepositBlocks float32 = 10
+	PropDepositBlocks float32 = 1
```

Comment:
^ this is the vote speed up. This is possible now because we vote across all validators at once and no longer have to wait for confirmation on each one